packer/ansible/roles/images/tasks/main.yml

```diff
@@ -20,3 +20,11 @@
    - monorail-efi32-snponly.efi
    - monorail-efi64-snponly.efi
    - undionly.kpxe
```

Comment:
Thanks gents, I wasn't sure if we should keep the ipxe generated undionly.kpxe or not. I'm happy to drop it if we're not using it today. Will update this shortly with undionly.kpxe from iPXE removed